src/modules/translation/languages/en.json

```diff
@@ -148,7 +148,8 @@
   },
   "claim_ens_page": {
     "title": "Claim a Name",
-    "subtitle": "You can claim names and assign them to Profiles or Land, and they can also be used as a quick link to access the world.",
+    "subtitle": "A name assignment allows you have a unique URL for your parcel with the following format: http://name.dcl.eth.link. {separator} Once users open http://name.dcl.eth.link, they will be redirected to your parcel in the {dclWorldLink}",
```

Comment:
I like this text! two things tho:
- Maybe we can lead with `Assigning a name` instead of `A name assignment` to give emphasis to the action 
- The link we create ends on `.link`? I was sure it was just `http://name.dcl.eth`

Reply:
Thanks for the new text!!! It is fixed!